packages/MarkdownEditor-Tests.package/MarkdownCommentBlockTests.class/instance/testClosingTagInFirstLineCloses.st

```diff
@@ -0,0 +1,6 @@
+tests - extendability
+testClosingTagInFirstLineCloses
+
+	| commentBlock |
+	commentBlock := MarkdownCommentBlock newFrom: '<!-- MiniMetro -->', Character cr.
```

Comment:
'This is the granularity where I prefer the self-explaining version. I honestly laughed about the block structure tests, but here it feels off.

```suggestion
	commentBlock := MarkdownCommentBlock newFrom: '<!-- Comment closed at the same line -->', Character cr.